mne_bids_pipeline/config.py

```diff
@@ -3788,19 +3796,16 @@ def import_er_data(
 def import_rest_data(
     *,
     cfg: SimpleNamespace,
-    subject: str,
-    session: Optional[str] = None
+    bids_path_in: BIDSPath,
```

Comment:
I think this was just not being used/tested at all on `main`. `import_experimental_data` cannot be called without a `bids_path_in` argument, and it was below...